packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html

```diff
@@ -67,6 +65,52 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
                                 data-title="{{ 'telecom_pack_migration_confirm_header_total' | translate }}"
                             ></td>
                         </tr>
+                        <tr data-ng-if="$ctrl.process.selectedOffer.firstYearPromo.value > 0">
+                            <th
+                                scope="row"
+                                data-translate="telecom_pack_migration_confirm_first_year_promo"
+                                data-title="{{ 'telecom_pack_migration_confirm_header_entitled' | translate }}"
+                            ></th>
+                            <td
+                                class="text-price text-right"
+                                data-ng-bind="$ctrl.getReductionPriceFirstYear().text"
+                                data-title="{{ 'telecom_pack_migration_confirm_header_price' | translate }}"
+                            ></td>
+                            <td
+                                data-ng-bind="1"
+                                class="text-right"
+                                data-title="{{ 'telecom_pack_migration_confirm_header_quantity' | translate }}"
+                            ></td>
```

Comment:
```suggestion
                            <td
                                class="text-right"
                                data-title="{{ 'telecom_pack_migration_confirm_header_quantity' | translate }}"
                            >1</td>
```
If the value is static no need for `ng-bind`